tests/ml_commons/test_ml_commons_client.py

```diff
@@ -33,7 +33,9 @@
 MODEL_CONFIG_FILE_NAME = "ml-commons_model_config.json"
 
 TEST_FOLDER = os.path.join(PROJECT_DIR, "test_model_files")
-TESTDATA_SYNTHETIC_QUERY_ZIP = os.path.join(PROJECT_DIR, "..", "synthetic_queries.zip")
+TESTDATA_SYNTHETIC_QUERY_ZIP = os.path.join(
```

Comment:
Looks like you are changing the path here? Can we revert this?

Reply:
Oh, I don't remember changing it. Will fix.